message/job.py

```diff
@@ -0,0 +1,71 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+""" Represents the messages passed between AMQ queues"""
+import json
+
+
+# pylint:disable=too-many-instance-attributes
+class Message:
+    """
+    A class that represents an AMQ Message, these can be both serialised and un-serialised
```

Comment:
A class that represents an AMQ **Message.**
Messages can be both serialised and unserialised
    for sending messages to and from AMQ

_(2 sentences rather than 1)_